plugins/db-postgres/src/constants.ts

```diff
@@ -23,7 +23,7 @@ export const updateDockerComposeProperties: CreateServerDockerComposeParams["upd
       services: {
         server: {
           environment: {
-            DB_URL: "postgres://${DB_USER}:${DB_PASSWORD}@db:5433",
+            DB_URL: "postgres://${DB_USER}:${DB_PASSWORD}@db:5432",
```

Comment:
I think we should use the dynamic values here like:
`DB_URL: "postgres://${DB_USER}:${DB_PASSWORD}@db:${DB_PORT}"`

And we would probably need the same approach for both, `server` and `migrate` containers